src/components/FullResourcePage.tsx

```diff
@@ -0,0 +1,170 @@
+import axios from "axios";
+import { useState } from "react";
+import { useParams } from "react-router";
+import { BackendURL } from "../utils/backendURL";
+import useFetchResources from "../utils/useFetchResources";
+import "./FullResourcePage.css";
+import useFetchComments from "../utils/useFetchComments";
+import { ToastContainer, toast } from "react-toastify";
+import "react-toastify/dist/ReactToastify.css";
+
+//--------------------------------------------------------------------------------------JSX Element declaration
+export default function FullResourcePage(): JSX.Element {
+  const { resources } = useFetchResources();
+
+  const [commentSubmit, setCommentSubmit] = useState({
+    user_id: 0,
+    comment_text: "",
+  });
+
+  //--------------------------------------------------------------------------------------Defining selected resource
+  const { id } = useParams();
+  const resource = resources.find((resource) => {
+    if (id) {
+      return resource.resource_id === parseInt(id);
+    }
+    return false;
+  });
+  const user = localStorage.getItem("user");
+  //--------------------------------------------------------------------------------------GET comments from SERVER
+  const { commentList, updateComments } = useFetchComments(id);
+
+  const postCommentToServer = async (user_id: number, comment_text: string) => {
+    if (comment_text.length > 0 && id) {
+      try {
+        await axios.post(BackendURL + "comments", {
+          resource_id: id,
+          user_id: user_id,
+          comment_text: comment_text,
+        });
+        updateComments();
+      } catch (error) {
+        console.log("error from post");
+      }
+    } else {
+      alert("you must write something before you submit!");
+    }
+  };
+
+  const handleCommentSubmit = async (e: React.FormEvent<HTMLFormElement>) => {
+    if (user) {
+      postCommentToServer(JSON.parse(user).user_id, commentSubmit.comment_text);
+    } else {
+      signIn();
+    }
+  };
+
+  const handleDeleteComment = async (comment_id: number) => {
+    console.log("deleting comment", comment_id);
```

Comment:
like the use of console logs to test code however unsure if they should be kept in ?